logback-contrib/pom.xml

```diff
@@ -0,0 +1,169 @@
+        <junit.version>4.8.2</junit.version>
+        <hamcrest.version>2.0.0.0</hamcrest.version>
+
+    </properties>
+
+    <dependencies>
+        <!-- Test Dependencies for _all_ children modules: -->
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-junit</artifactId>
+            <version>${hamcrest.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <dependencyManagement>
+        <dependencies>
+
+            <!-- Pool of dependencies available to any sub-module, but not included automatically.
+                They must be explicitly referenced in the children POMs, but _don't_ include
+                version numbers in the child POMs.  Define them here only to guarantee version
+                compatibility across children modules! -->
+
+            <dependency>
+                <groupId>org.apache.james</groupId>
+                <artifactId>james-project</artifactId>
+                <version>3.9.0-SNAPSHOT</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+
+            <!-- Logback Extensions sub-modules dependency .jars (alphabetized based on artifactId for easy scanning please!): -->
+            <dependency>
+                <groupId>ch.qos.logback</groupId>
+                <artifactId>logback-access</artifactId>
+                <version>${logback.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>ch.qos.logback</groupId>
+                <artifactId>logback-core</artifactId>
+                <version>${logback.version}</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>3.5.0</version>
+                <executions>
+                    <execution>
+                        <id>enforce-dependency-versions</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireSameVersions>
+                                    <plugins>
+                                        <plugin>ch.qos.logback:logback-classic</plugin>
+                                        <plugin>ch.qos.logback:logback-core</plugin>
+                                        <plugin>ch.qos.logback:logback-access</plugin>
+                                    </plugins>
+                                </requireSameVersions>
+                            </rules>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
```

Comment:
Maybe locate this into tmail-backend/tmail-backend as it might be there that logback diverges between james project.